federation-js/src/composition/validate/sdl/__tests__/uniqueTypeNamesWithFields.test.ts

```diff
@@ -147,6 +167,16 @@ describe('UniqueTypeNamesWithFields', () => {
         Array [
           Object {
             "code": "VALUE_TYPE_INPUT_VALUE_MISMATCH",
+            "locations": Array [
```

Comment:
Should these locations point to the arg value's type in each service?

Reply:
This one is not easy, since we don't return the input value nodes, just the names from `diffTypeNodes`